src/appModel.ts

```diff
@@ -26,12 +26,20 @@ export class AppModel {
     }
 
     StartWatching() {
+        const compileOnWatch = Helper.getConfigSettings<boolean>('compileOnWatch');
+
         if (this.isWatching) {
             WindowPopout.Inform('Already watching...');
         }
         else {
             this.isWatching = !this.isWatching;
-            this.compileAllFiles();
+            StatusBarUi.working();
```

Comment:
Remove this. [ref](https://github.com/glenn2223/vscode-live-sass-compiler/pull/3#issuecomment-748591053)